tests/test_api.py

```diff
@@ -556,12 +558,21 @@ def test_get_wiht_filter_nested_rewrites(self):
         content = json.loads(response.content.decode('utf-8'))
         self.assertEqual(1, len(content['users']))
 
-    def test_get_with_filter_invalid(self):
+    def test_get_with_filter_nonexistent_field(self):
         # Filtering on non-existent field should return 400
         url = '/users/?filter{foobar}=1'
         response = self.client.get(url)
         self.assertEqual(400, response.status_code)
 
+    def test_get_with_filter_invalid_data(self):
+        User.objects.create(
+            name='test',
+            date_of_birth=datetime.datetime.utcnow()
+        )
+        url = '/users/?filter{date_of_birth.gt}=0&filter{date_of_birth.lt}=0'
+        response = self.client.get(url)
+        self.assertEqual(400, response.status_code)
```

Comment:
Can you add an assertion for the response? Would be nice to know what sort of error message you would see if this happened